backend/tasks.py

```
@@ -88,11 +94,16 @@ async def create_grievance_from_issue_background(issue_id: int):
     except Exception as e:
-        logger.error(f"Error creating grievance from issue {issue_id}: {e}", exc_info=True)
+        logger.error(
+            f"Error creating grievance from issue {issue_id}: {e}", exc_info=True
+        )
     finally:
         db.close()
 
-def send_status_notification(issue_id: int, old_status: str, new_status: str, notes: str = None):
+
+def send_status_notification(
+    issue_id: int, old_status: str, new_status: str, notes: str = None
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)